airflow/utils/cli.py

```diff
@@ -306,11 +298,13 @@ class ColorMode:
     AUTO = "auto"
 
 
-def should_use_colors(args) -> bool:
+def should_use_colors(args_or_color) -> bool:
     """Processes arguments and decides whether to enable color in output"""
-    if args.color == ColorMode.ON:
+    # args.color is from argparse, Click CLI will pass in the color directly
+    color = args.color if hasattr(args, 'color') else args_or_color
```

Comment:
Backwards compatibility with argparse. I included the comment so we can grep for argparse and simplify this once we no longer need argparse.